tools/pd-heartbeat-bench/main.go

```diff
@@ -46,9 +47,12 @@ import (
 )
 
 const (
-	bytesUnit            = 8 * units.MiB
-	keysUint             = 8 * units.KiB
-	queryUnit            = 1 * units.KiB
+	bytesUnit            = 128
```

Comment:
When we use it, we need to note that item of tikv heartbeat is the cumulative, which is used for hotspot determination is the rate.

For example, if the hotKeysUint is 256, the calculated rate is 25.6 for reads and 4.2 for writes, which is lower than the threshold for the hot side, so it won't be treated as a hot region.